.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+    paths-ignore:
+    - "nuxt/**"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    outputs:
+      new_release_version: ${{ steps.semantic.outputs.new_release_version }}
+      new_release_published: ${{ steps.semantic.outputs.new_release_published }}
+  
+    steps:
+      - uses: actions/checkout@v3
+      - run: yarn
+      - uses: cycjimmy/semantic-release-action@e1fe1fc00a3729593e87efb2f88475de76d64a24
+        id: semantic
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It will first create a new GitHub Release, with release notes and a CHANGELOG file.

It will decide on the version number based on commit messages.